src/infrastructure/ajv.ts

```diff
@@ -53,6 +53,24 @@ export function parseJsonOfSchema<T>(
 	}
 }
 
+export function parseJsonOfSchemaFromObject<T>(
```

Comment:
**issue:** This method can be unnecessary -- consider using the existing `assertSchema` instead.